test/Dialect/FIRRTL/imconstprop.mlir

```diff
@@ -589,3 +589,16 @@ firrtl.circuit "ForwardRef" {
     firrtl.strictconnect %a, %0 : !firrtl.uint<1>
   }
 }
+
+firrtl.circuit "Top"  {
+  firrtl.module @Top(in %in: !firrtl.uint<1>, out %out: !firrtl.uint<1>) attributes {annotations = [{class = "firrtl.transforms.NoDedupAnnotation"}]} {
```

Comment:
```suggestion
  firrtl.module @Top() {
```

Reply:
Consider using a more descriptive module name to indicate what this test is checking for, or add a comment indicating such / mentioning the issue?

Reply:
Thanks @dtzWill , one more copy paste oversight, fixed the lit tests.